homeassistant/components/hyperion/config_flow.py

```diff
@@ -402,6 +418,18 @@ async def async_step_confirm(
             return self.async_abort(reason="no_id")
 
         await self.async_set_unique_id(hyperion_id, raise_on_progress=False)
```

Comment:
This returns an existing entry or None. We can use this and remove the need for storing the entry id.